### PR TITLE
New package: sopwith-1.8.4

### PR DESCRIPTION
**Pull request by beefcurtains**
Upstream package can't be cross compiled, so I made a patch that fixes it.

### CODE REVIEW

File: srcpkgs/sopwith/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'sopwith'
+pkgname=sopwith
+version=1.8.4
+revision=1
+build_style=gnu-configure
+depends="SDL"
+makedepends="${depends}-devel"
+hostmakedepends="autoconf automake ${makedepends}"
```
Comment (xtraeme) [inline srcpkgs/sopwith/template]: This is completely wrong... you only really need `SDL-devel` in `makedepends`.

Comment (beefcurtains) [inline srcpkgs/sopwith/template]: That'll fail when autoreconf is run.

Comment (xtraeme) [inline srcpkgs/sopwith/template]: In this case just drop `depends`. It's unnecessary with `xbps-src`.

Comment (beefcurtains) [inline srcpkgs/sopwith/template]: Thank you :heart: and sorry, my bad!

File: srcpkgs/sopwith/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'sopwith'
+pkgname=sopwith
+version=1.8.4
+revision=1
+build_style=gnu-configure
+makedepends="${depends}-devel"
```
Comment (xtraeme) [inline srcpkgs/sopwith/template]: This is wrong...

Comment (beefcurtains) [inline srcpkgs/sopwith/template]: I know, already fixed.

File: srcpkgs/sopwith/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'sopwith'
+pkgname=sopwith
+version=1.8.4
+revision=1
+build_style=gnu-configure
+makedepends="SDL-devel"
+hostmakedepends="autoconf automake ${makedepends}"
```
Comment (xtraeme) [inline srcpkgs/sopwith/template]: `autoconf` is redundant because it's a dependency of `automake`

Comment (beefcurtains) [inline srcpkgs/sopwith/template]: Sorry about that, should I also rebase the whole thing?

Comment (xtraeme) [inline srcpkgs/sopwith/template]: it is ok

**Pull request merged**